### PR TITLE
Change 2.0-alpha1 to 2.0-rc1.

### PR DESCRIPTION
**Pull request by dblock**
Following https://github.com/opensearch-project/opensearch-build/pull/1863, change 2.0-alpha1 to 2.0-rc1.

### CODE REVIEW

**Comment by peternied** [comment]
@dblock This is a chicken v. egg build problem.  The security plugin uses the OpenSearch min tar to make sure it can be installed.  Until we've got a rc1 version of OpenSearch min our CI check won't pass.

Option 1) remove security plugin from the manifest.  Cleanest approach for build safety with the tradeoff of consuming a lot of time during this transition phase.

Option 2) bypass the CI check and cross fingers that the new build doesn't have any unexpected issues.

My preference is towards option 1 with the understanding it will slow down the whole rollout.  This will ensure only correctly building plugins are included in the build, avoiding an utterly broken build.

Note; These problems are magnified in security-dashboards-plugin as it depends on a full distribution build for its CI checks.

**Comment by DarshitChanpura** [comment]
> @dblock This is a chicken v. egg build problem. The security plugin uses the OpenSearch min tar to make sure it can be installed. Until we've got a rc1 version of OpenSearch min our CI check won't pass.
> 
> Option 1) remove security plugin from the manifest. Cleanest approach for build safety with the tradeoff of consuming a lot of time during this transition phase.
> 
> Option 2) bypass the CI check and cross fingers that the new build doesn't have any unexpected issues.
> 
> My preference is towards option 1 with the understanding it will slow down the whole rollout. This will ensure only correctly building plugins are included in the build, avoiding an utterly broken build.
> 
> Note; These problems are magnified in security-dashboards-plugin as it depends on a full distribution build for its CI checks.

I have created a PR to address option 1. https://github.com/opensearch-project/opensearch-build/pull/1977

**Comment by dblock** [comment]
Once there's a successful distribution rc1 build CI will turn green here, let's retry later today.

**Comment by DarshitChanpura** [comment]
Thank you @dblock !

File: release-notes/opensearch-security.release-notes-1.0.0.0-beta1.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Compatible with OpenSearch 1.0.0-beta1
 * Remove code setting the value for cluster.routing.allocation.disk.threshold_enabled ([#1067](https://github.com/opensearch-project/security/pull/1067))
 * Rename for OpenSearch ([#1126](https://github.com/opensearch-project/security/pull/1126))
 * Fix CI ([#1131](https://github.com/opensearch-project/security/pull/1131))
-* Consume OpenSearch 1.0.0-alpha1 ([#1132](https://github.com/opensearch-project/security/pull/1132))
+* Consume OpenSearch 1.0.0-rc1 ([#1132](https://github.com/opensearch-project/security/pull/1132))
```
Comment (peterzhuamazon) [inline release-notes/opensearch-security.release-notes-1.0.0.0-beta1.md:22]: You should not change this @dblock

**Review by peterzhuamazon — CHANGES_REQUESTED** [review]
Wrong edit changelog

**Review by DarshitChanpura — APPROVED** [review]

**Review by peterzhuamazon — APPROVED** [review]

**Pull request merged**